applications/conceptual-model-editor/src/diagram/diagram-controller.ts

```diff
@@ -456,6 +456,13 @@ const createActions = (
         focusNodeAction(reactFlow, node);
       }
     },
+    fitToView(identifiers) {
+      console.log("Diagram.fitToView", { identifiers });
```

Comment:
Do we need this log here? Seems like it was used for debug during development of  a particular feature, perhaps not needed anymore?